acm/ivy.xml

```diff
@@ -33,7 +33,10 @@
     <dependency org="com.seaglasslookandfeel" name="seaglasslookandfeel" rev="0.2.0-JAVA8"/>
     <dependency org="org.swinglabs" name="swingx" rev="1.6.1"/>
     <dependency org="args4j" name="args4j" rev="2.0.16"/>
-    <dependency org="commons-io" name="commons-io" rev="2.4"></dependency>
-    <dependency org="net.sf.opencsv" name="opencsv" rev="2.3"></dependency>
+    <dependency org="commons-io" name="commons-io" rev="2.4"/>
+    <dependency org="net.sf.opencsv" name="opencsv" rev="2.3"/>
+    <dependency org="org.powermock" name="powermock-api-mockito" rev="1.6.4" />
```

Comment:
That's a great idea. I haven't been able to make Ivy/Ant make it work, though. It does seem to be just what Ivy should be good for.

I'll continue to work on that, but I may deploy it initially without. It is "only" a little under 2MB.